src/appengine/handlers/cron/backup.py

```diff
@@ -38,17 +38,17 @@ def _datastore_client():
   return googleapiclient.discovery.build('datastore', 'v1')
 
 
-class Handler(base_handler.Handler):
+class Handler(base_handler_flask.Handler):
   """Handler for triggering the backup URL."""
 
-  @handler.check_cron()
+  @handler_flask.cron()
   def get(self):
     """Handle a cron job."""
     backup_bucket = local_config.Config(
         local_config.PROJECT_PATH).get('backup.bucket')
     if not backup_bucket:
       logs.log('No backup bucket is set, skipping.')
-      return
+      return 'OK'
```

Comment:
Remove 'OK'

Reply:
Here actually I added 'OK'  because we are returning a tuple in the same function, if we don't return anything here there will be a lint check fail of inconsistent-return-statements